tests/integration/conftest.py

```diff
@@ -57,6 +57,14 @@ def synapse_image_fixture(pytestconfig: Config):
     return synapse_image
 
 
+@pytest_asyncio.fixture(scope="module", name="synapse_nginx_image")
+def synapse_nginx_image_fixture(pytestconfig: Config):
+    """Get value from parameter synapse-nginx-image."""
+    synapse_nginx_image = pytestconfig.getoption("--synapse-nginx-image")
```

Comment:
Perhaps `--synapse-nginx-image` should be a constant since I think it is in 3 places now?

Reply:
I only found 2 places where --synapse-nginx-image appears, same for other integration test parameters.

Reply:
It appears in `tests/conftest.py` and I'm guessing it should appear twice in this function, once on this line and once on the next line, though that says `--synapse-image` at the moment which I'm guessing should say `--synapse-nginx-image`

Reply:
I see. In this case, I changed the params to be constants then.